src/www/ui/template/ui-export-list.html.twig

```diff
@@ -74,6 +75,31 @@
             exclude all files and directories containing the substring 'mac'.
           </label>
         </li>
+        <li>
+          <span>
+            {{ "Consolidate Results?"|trans }}
+          </span>
+          <ul>
+            <li>
+              <lable>
+                <input type="radio" name="consolidate" value="rawResult" checked="checked" {% if rawResult %}checked{% endif %}>
```

Comment:
You might not wanna hard code `checked="checked"` in case `perFile` or `perDirectory` is true.

Try setting `$formVars["rawResult"] = true;` if `$consolidateLicenses` and `$consolidatePerDirectory` are `false` so that Raw result per file is checked on first load (or if nothing is selected by user).

Reply:
Updated as suggested. Thanks @GMishx .